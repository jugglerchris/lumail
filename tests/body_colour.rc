col = body_colour()
io.write(col.."\n")
body_colour('blue')
col = body_colour()
io.write(col.."\n")
