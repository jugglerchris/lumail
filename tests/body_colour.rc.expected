white
blue
Exit: 0
